pyanaconda/modules/common/constants/namespaces.py

```diff
@@ -96,12 +86,22 @@
     "Baz"
 )
 
+PAYLOADS_NAMESPACE = (
+    *MODULES_NAMESPACE,
+    "Payloads"
+)
+
+DNF_NAMESPACE = (
+    *PAYLOADS_NAMESPACE,
+    "DNF"
+)
+
 PAYLOAD_BASE_NAMESPACE = (
```

Comment:
What about using the names `PAYLOADS_PAYLOAD_NAMESPACE` and `PAYLOADS_SOURCE_NAMESPACE`? Is it too confusing? Btw, I am fine also with the name `PAYLOAD_NAMESPACE`.

Reply:
Really not sure. I'm thinking the same. It's really hard to find some good solution for payloads and payload...

That would be inconsistent with `PAYLOAD_LIVE_OS` or similar and I don't think we want to rename payload to `PAYLOADS_LIVE_OS`.

Maybe the best solution would be to have `PAYLOAD_BASE_NAMESPACE` and `PAYLOAD_SOURCE_BASE_NAMESPACE` or just `PAYLOAD_BASE_NAMESPACE`?

Reply:
I don't like `BASE` in the name, because I feel we usually use it in a little different context. What about `PAYLOAD_GENERAL`? Anyway, feel free to merge the pull request as it is. It is not so important.